### DISCUSSION TITLE
deploy failed

### DISCUSSION THREAD

**Discussion by elenashcherbinina** [hexlet-volunteers/runit#376] [Q&A] [answered]
Упал деплой на рендер с такой ошибкой
 
`Sep 14 12:39:14 PM  ==> Starting service with 'make db-migrate start-prod'
Sep 14 12:39:14 PM  make -C backend db-migrate
Sep 14 12:39:14 PM  make[1]: Entering directory '/opt/render/project/src/backend'
Sep 14 12:39:14 PM  npm run typeorm -- migration:run -d src/data-source
Sep 14 12:39:17 PM  
Sep 14 12:39:17 PM  > backend@1.0.0 typeorm
Sep 14 12:39:17 PM  > typeorm-ts-node-commonjs migration:run -d src/data-source
Sep 14 12:39:17 PM  
Sep 14 12:40:05 PM  Error during migration run:
Sep 14 12:40:05 PM  Error: getaddrinfo ENOTFOUND dpg-ck1ctteru70s73ck694g-a
Sep 14 12:40:05 PM      at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:118:26) {
Sep 14 12:40:05 PM    errno: -3008,
Sep 14 12:40:05 PM    code: 'ENOTFOUND',
Sep 14 12:40:05 PM    syscall: 'getaddrinfo',
Sep 14 12:40:05 PM    hostname: 'dpg-ck1ctteru70s73ck694g-a'
Sep 14 12:40:05 PM  }
Sep 14 12:40:05 PM  npm ERR! Lifecycle script `typeorm` failed with error:
Sep 14 12:40:05 PM  npm ERR! Error: command failed
Sep 14 12:40:05 PM  npm ERR!   in workspace: backend@1.0.0
Sep 14 12:40:05 PM  npm ERR!   at location: /opt/render/project/src/backend
Sep 14 12:40:05 PM  make[1]: *** [Makefile:7: db-migrate] Error 1
Sep 14 12:40:05 PM  make[1]: Leaving directory '/opt/render/project/src/backend'
Sep 14 12:40:05 PM  make: *** [Makefile:7: db-migrate] Error 2
Sep 14 12:41:37 PM  ==> Using Node version 20.6.1 via /opt/render/project/src/package.json
Sep 14 12:41:37 PM  ==> Docs on specifying a Node version: https://render.com/docs/node-version
Sep 14 12:41:37 PM  ==> Starting service with 'make db-migrate start-prod'
Sep 14 12:41:37 PM  make -C backend db-migrate
Sep 14 12:41:37 PM  make[1]: Entering directory '/opt/render/project/src/backend'
Sep 14 12:41:37 PM  npm run typeorm -- migration:run -d src/data-source
Sep 14 12:41:41 PM  
Sep 14 12:41:41 PM  > backend@1.0.0 typeorm
Sep 14 12:41:41 PM  > typeorm-ts-node-commonjs migration:run -d src/data-source
Sep 14 12:41:41 PM  
Sep 14 12:42:34 PM  Error during migration run:
Sep 14 12:42:34 PM  Error: getaddrinfo ENOTFOUND dpg-ck1ctteru70s73ck694g-a
Sep 14 12:42:34 PM      at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:118:26) {
Sep 14 12:42:34 PM    errno: -3008,
Sep 14 12:42:34 PM    code: 'ENOTFOUND',
Sep 14 12:42:34 PM    syscall: 'getaddrinfo',
Sep 14 12:42:34 PM    hostname: 'dpg-ck1ctteru70s73ck694g-a'
Sep 14 12:42:34 PM  }
Sep 14 12:42:35 PM  npm ERR! Lifecycle script `typeorm` failed with error:
Sep 14 12:42:35 PM  npm ERR! Error: command failed
Sep 14 12:42:35 PM  npm ERR!   in workspace: backend@1.0.0
Sep 14 12:42:35 PM  npm ERR!   at location: /opt/render/project/src/backend
Sep 14 12:42:35 PM  make[1]: *** [Makefile:7: db-migrate] Error 1
Sep 14 12:42:35 PM  make[1]: Leaving directory '/opt/render/project/src/backend'
Sep 14 12:42:35 PM  make: *** [Makefile:7: db-migrate] Error 2
`

**Comment by elenashcherbinina** [comment]
Переменные добавила, запустила снова деплой, но он упал с той же ошибкой, что-то связанное с миграцией. Прилагаю скрины.
<img width="809" alt="Снимок экрана 2023-09-14 в 12 41 13" src="https://github.com/hexlet-rus/runit/assets/108113888/5dad9b82-2b24-40ca-9809-aa09164fbcfc">

<img width="803" alt="Снимок экрана 2023-09-15 в 14 29 16" src="https://github.com/hexlet-rus/runit/assets/108113888/ce052c25-22b3-4f46-810f-d7eaf7e67291">

<img width="759" alt="Снимок экрана 2023-09-15 в 14 29 22" src="https://github.com/hexlet-rus/runit/assets/108113888/881c6a6a-5253-4905-968a-551af8b6409e">

Локально развернула проект без проблем.

**Reply by fey** [reply, marked as answer]
пишет о том, что не получилось подключиться к БД.
роверьте, что у вас база данных и приложение находятся в одном регионе. 
Плюс что вы скопировали именно internal database url

**Comment by elenashcherbinina** [comment]
Николай спасибо! Дело было именно в разных регионах базы и самого сервиса, сделала один регион, все получилось: https://elenashcherbinina-runit.onrender.com/

Единственное, сейчас падает регистрация через гитхаб, попробую сегодня еще раз через oAuth.
Меня кстати удалили из мм.

**Reply by fey** [reply]
по ММ попрошу, чтобы вернули
